go.mod

```diff
@@ -26,12 +26,12 @@ require (
 	github.com/spf13/pflag v1.0.3
 	github.com/tektoncd/pipeline v0.3.1
 	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80
-	golang.org/x/tools v0.0.0-20190506145303-2d16b83fe98c // indirect
-	halkyon.io/api v1.0.0-beta.5
+	golang.org/x/tools v0.0.0-20190506145303-2d16b83fe98c
+	halkyon.io/api v1.0.0-beta.7
 	k8s.io/api v0.0.0-20190725062911-6607c48751ae
 	k8s.io/apimachinery v0.0.0-20190719140911-bfcf53abc9f8
 	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/kubernetes v1.14.5 // indirect
+	k8s.io/kubernetes v1.14.5
```

Comment:
- Is is the same version as used within the API ? 
- Should we align this k8s version with what is defined within the doc ? 
- Are we doing our tests against k8s 1.14 ?

Reply:
I'm actually not sure where this is coming from… :(